bayesian_cnn_prometheus/constants.py

```diff
@@ -34,15 +34,19 @@ class Paths:
     REFERENCE_SEGMENTATIONS_DIR = 'REFERENCE_SEGMENTATIONS'
     REFERENCE_SEGMENTATIONS_PATH = DATA_DIR / REFERENCE_SEGMENTATIONS_DIR
     REFERENCE_SEGMENTATION_FILE_PATTERN = 'LUNGS_IMG_{}.{}'
-    REFERENCE_SEGMENTATION_FILE_PATTERN_PATH = REFERENCE_SEGMENTATIONS_PATH / REFERENCE_SEGMENTATION_FILE_PATTERN
+    REFERENCE_SEGMENTATION_FILE_PATTERN_PATH = REFERENCE_SEGMENTATIONS_PATH / \
+        REFERENCE_SEGMENTATION_FILE_PATTERN
 
     RESULTS_DIR = 'RESULTS'
     RESULTS_PATH = DATA_DIR / RESULTS_DIR
 
     PREDICTIONS_FILE_PATTERN = 'PREDICTIONS_{}.{}'
     PREDICTIONS_FILE_PATTERN_PATH = RESULTS_PATH / PREDICTIONS_FILE_PATTERN
 
-    SUMMARY_FILE_PATTERN = 'SUMMARY_ID_{}_SLICE_{}.{}'
+    VARIANCE_FILE_PATTERN = 'VARIANCE_{}.{}'
+    VARIANCE_FILE_PATTERN_PATH = RESULTS_PATH / VARIANCE_FILE_PATTERN
+
+    SUMMARY_FILE_PATTERN = 'SUMMARY_ID_{}.{}'
```

Comment:
Teraz zapisujemy cały przewidziany skan, zamiast zapisywać tylko jeden slice

Reply:
W sumie może prediction byłoby lepiej niż summary? W sensie to jest średnia predykcja, a słowo "podsumowanie" niekoniecznie na to wskazuje